tpgtools/overrides/compute/samples/networkfirewallpolicyrule/global.tf.tmpl

```diff
@@ -0,0 +1,41 @@
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name = "{{policy}}"
+  project = "{{project}}"
+  description = "Sample global network firewall policy"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
```

Comment:
`enable_logging` and `disabled` don't appear in tests. Could we add them and test transitioning one of them from `true` -> `false`? Setting booleans to `false` is often a cause of failures in updates.

Reply:
good catch, will do